app/views/devise/confirmations/new.html.erb

```diff
@@ -1,15 +1,15 @@
-<h2>Resend confirmation instructions</h2>
+<h1 class="h4 text-gray-900 mb-4">Resend confirmation instructions</h1>
 
 <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "Email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "Email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), class: "form-control", required: true  %>
```

Comment:
Always use single quotes in ruby code when no string interpolation is being done
```suggestion
    <%= f.email_field :email, autofocus: true, autocomplete: 'Email', value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), class: 'form-control', required: true  %>
```

Reply:
done